### PR TITLE
https://developer.salesforce.com/forums/?id=906F00000008zoyIAA

### PR DESCRIPTION
**Pull request by hartnady**
https://developer.salesforce.com/forums/?id=906F00000008zoyIAA

### CODE REVIEW

**Comment by alexed1** [comment]
Thanks for this, Mark. I think I have a decent sense of what's going on here, but can you summarize the change briefly? You don't see any risks to removing that comma?

**Comment by hartnady** [comment]
The test class fails on recent versions of salesforce and it passes when
removing the comma. The only other thing I’d suggesting is forked logic.
E.g.  If API version less than X add comma, Else remove it.

**Review by alexed1 — APPROVED** [review]

**Comment by alexed1** [comment]
When I tried to package with this change, I got: 
ERROR running force:package:version:create:  Multiple errors occurred:
(1) Apex Test Failure: Class.ExecuteSOQL.getFormattedValue: line 134, column 1
Class.ExecuteSOQL.putFormattedValues: line 103, column 1
Class.ExecuteSOQL.replaceWithFormattedValues: line 77, column 1
Class.ExecuteSOQL.execute: line 14, column 1
Class.ExecuteSOQLTest.testExecuteEnglishUpperCasePM: line 13, column 1 System.TypeException: Invalid date/time:  04/12/2020 03:24 PM
(2) Apex Test Failure: Class.ExecuteSOQL.getFormattedValue: line 134, column 1
Class.ExecuteSOQL.putFormattedValues: line 103, column 1
Class.ExecuteSOQL.replaceWithFormattedValues: line 77, column 1
Class.ExecuteSOQL.execute: line 14, column 1
Class.ExecuteSOQLTest.testExecuteSOQLMonthDayCommaYear: line 30, column 1 System.TypeException: Invalid date/time:  04/12/2020 12:00 am
